File: q6.cpp
#include <iostream>
using namespace std;

class num
{
public:
    inline int square(int a)
    {
        return a * a;
    }

    inline int cube(int a)
    {
        return a * a * a;
    }
};

int main()
{
    num n;
    int num;

    cout << "Enter any number: ";
    cin >> num;

    cout << "Square: " << n.square(num) << endl;
    cout << "Cube: " << n.cube(num) << endl;

    return 0;
}

File: java/LabWork/q6.java
package LabWork;

import java.util.Scanner;

public class q6 {
    public static void main(String[] args){
    int num , count = 0;

    Scanner s = new Scanner(System.in);

    System.out.print("Enter any number: ");
    num = s.nextInt();

    for(int i = 2; i<num; i++){
        if(num % i == 0){
            count++;
            break;
        }
    }

    if(count == 0)
        System.out.println(num + " is a prime number");
    else
       System.out.println(num + " is not a prime number"); 
    
    s.close();
}
}